src/molecule/test/conftest.py

```diff
@@ -138,3 +139,13 @@ def reset_pytest_vars(monkeypatch):
     for var_name in tuple(os.environ):
         if var_name.startswith("PYTEST_"):
             monkeypatch.delenv(var_name, raising=False)
+
+
+@pytest.fixture(autouse=True)
+def block_on_serial_mark(request):
+    # https://github.com/pytest-dev/pytest-xdist/issues/385
+    if request.node.get_closest_marker("serial"):
+        with FileLock(".tox/semaphore.lock"):
```

Comment:
Do we have guarantees that the `.tox` directory will always be there when we run the tests? Or, will this create the missing directory if it is missing?

Reply:
Fixed.